### PR TITLE
Readme: updated readme to reflect new changes

### PR DESCRIPTION
**Pull request by mcharanrm**
#### Highlights of the PR

Updated Readme to reflect the new changes
1. Launch/Destroy Plain IBM cloud instances
2. Launch/Destroy IBM cloud instances with Tower Deployment
3. Launch/Destroy IBM cloud instances with Tower Deployment + Isolated Instances

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,20 +139,56 @@ ansible_ssh_private_key_file=conf/towerperf_id_rsa
...
+
+Run the `ansible-playbook` command with following `EXTRA` cmdline arguments
+
+    ansible-playbook cleanup.yml -e ibmcloud_vsi_count=2 \
+                                 -e ibmcloud_vpc_name_prefix='perf-scale-test'
+
+
+Looking for Tower deployment with Isolated Nodes
+------------------------------------------------
+    ansible-playbook create.yml -e install_iso=True
```
Comment (jainnikhil30) [inline README.md:172]: for the consistency with other examples, should we make it 
        `ansible-playbook create.yml -e ibmcloud_vsi_count=2 \
                                     -e ibmcloud_vpc_name_prefix='perf-scale-test' \
                                     -e install_iso=True`

**Review by jainnikhil30 — COMMENTED** [review]
Looks good apart from that one suggested change.

File: README.md (hunk 1 above)
Comment (mcharanrm) [reply README.md:172]: yeah sure thing :+1: will push the changes in a bit

Comment (mcharanrm) [reply README.md]: https://github.com/jainnikhil30/ibm_cloud_tower_deploy/pull/10/commits/f17e4354e907fdda31f2f0d768319da2c917dc55 adds the above suggested changes

**Review by jainnikhil30 — APPROVED** [review]

**Pull request merged**